src/KurrentDB.SecondaryIndexing.LoadTesting/Assertions/DuckDb/DuckDbIndexingSummaryAssertion.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Kurrent, Inc and/or licensed to Kurrent, Inc under one or more agreements.
+// Kurrent, Inc licenses this file to you under the Kurrent License v1 (see LICENSE.md).
+
+using KurrentDB.SecondaryIndexing.LoadTesting.Observability;
+using KurrentDB.SecondaryIndexing.Storage;
+using static KurrentDB.SecondaryIndexing.Indexes.Category.CategorySql;
+using static KurrentDB.SecondaryIndexing.Indexes.EventType.EventTypeSql;
+
+namespace KurrentDB.SecondaryIndexing.LoadTesting.Assertions.DuckDb;
+
+public class DuckDbIndexingSummaryAssertion(DuckDbDataSource db): IIndexingSummaryAssertion {
+	public async ValueTask IndexesMatch(IndexingSummary summary) {
+		await AssertCategoriesAreIndexed(summary);
+		await AssertEventTypesAreIndexed(summary);
+	}
+
+	private ValueTask AssertCategoriesAreIndexed(IndexingSummary summary) {
+		var categories = db.Pool.Query<CategorySummary, GetCategoriesSummaryQuery>();
+
+		if (categories.All(c => summary.Categories.ContainsKey(c.Name)))
```

Comment:
should a check also be done to see if summary.Categories contains extra categories that were not in the DB? (same applies for event types below)